model/src/w3triamd.F90

```diff
@@ -2730,6 +2840,9 @@ SUBROUTINE SETUGIOBP ( )
 #endif
       ITMP = MAPSTA(1,:)
       CALL SET_IOBP(ITMP, IOBP) 
+!AR: what would be the right unit number? 
```

Comment:
Should we address what the right unit number should be before merging this code, or has the right unit number been selected and this comment can be safely deleted?

Reply:
Good one, I have now chosen 23456, normally we use some code that produces some dynamic file handler, is there any rule in ww3, how we should assign file handles? Comment deleted, some file handle set.